.github/workflows/ruby_tests.yml

```
@@ -11,7 +11,12 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby-version: ['2.7.4', '3.0.2', '3.1.3', '3.2.1']
+        ruby-version:
+          - '2.7.4'
+          - '3.0.2'
+          - '3.1.3'
+          - '3.2.1'
+          - '3.3.5'
     steps:
       - uses: actions/checkout@v2
```

Unrelated, but we should update this
```suggestion
      - uses: actions/checkout@v4
```

---

https://github.com/theforeman/foreman_maintain/pull/964